src/sentry/monitors/tasks.py

```diff
@@ -160,32 +157,22 @@ def try_monitor_tasks_trigger(ts: datetime, partition: int):
 
     # Track the delay from the true time, ideally this should be pretty
     # close, but in the case of a backlog, this will be much higher
-    total_delay = datetime.now().timestamp() - reference_ts
+    total_delay = datetime.now().timestamp() - slowest_part_ts
 
-    # TODO(epurkhiser): For now we will just log the slowest partition
-    # timestamp and in production we can validate the value moves forward
-    # correctly. It's likely this will be a minute behind the actual
-    # reference_ts since there will always be a sloest partition
-    logger.info(
-        "monitors.consumer.clock_tick_slowest_partition",
-        extra={"slowest_part_ts": slowest_part_ts},
-    )
+    tick = datetime.fromtimestamp(slowest_part_ts)
 
-    logger.info(
-        "monitors.consumer.clock_tick",
-        extra={"reference_datetime": str(reference_datetime)},
-    )
+    logger.info("monitors.consumer.clock_tick", extra={"reference_datetime": str(tick)})
     metrics.gauge("monitors.task.clock_delay", total_delay, sample_rate=1.0)
 
     # If more than exactly a minute has passed then we've skipped a
     # task run, report that to sentry, it is a problem.
-    if last_ts is not None and reference_ts > last_ts + 60:
+    if last_ts is not None and slowest_part_ts > last_ts + 60:
         with sentry_sdk.push_scope() as scope:
             scope.set_extra("last_ts", last_ts)
-            scope.set_extra("reference_ts", reference_ts)
+            scope.set_extra("slowest_part_ts", slowest_part_ts)
             sentry_sdk.capture_message("Monitor task dispatch minute skipped")
 
-    _dispatch_tasks(ts)
+    _dispatch_tasks(tick)
```

Comment:
This is subtly different.

Prior to this we were using the `ts` of when the kafka message was read. This was down to the microsecond. However, both tasks run from `_dispatch_tasks` ALREADY were trimming off the `second` and `microsecond` part of the datetime, so it was essentially `datetime.fromtimestamp(reference_ts)` before.

`reference_ts` is now `slowest_part_ts` and `tick` is `


We need this to be the time of the slowest partition when it actually ticked forward